src/BasePlatform.ts

```diff
@@ -25,6 +25,9 @@ import {CheckUpdatesPayload} from "./dispatcher/payloads/CheckUpdatesPayload";
 import {Action} from "./dispatcher/actions";
 import {hideToast as hideUpdateToast} from "./toasts/UpdateToast";
 
+export const HS_URL_LS_KEY = "mx_hs_url";
+export const IS_URL_LS_KEY = "mx_is_url";
```

Comment:
LS isn't super obvious at first, can we call this just `HS_URL_KEY` or `HS_URL_STORAGE_KEY`?

Also please avoid making IS more prominent in the app - it makes things feel like booleans.

Reply:
we do similar for WellKnown abbreviating into WK_KEY

> Also please avoid making IS more prominent in the app - it makes things feel like booleans.

any suggestions for a better way to express IdentityServer without spelling it out?

Reply:
I suppose the problem is less so the LS and more the use of two shorthands in the same name. To unblock this, `HOMESERVER_URL_KEY` and `ID_SERVER_URL_KEY` are probably fine (I don't think we need to mention the storage medium)